charts/fulcrum/values.yaml

```
@@ -0,0 +1,73 @@
+secrets:
+  create: true
+
+image:
+  repository: cculianu/fulcrum
+  pullPolicy: IfNotPresent
+  tag: v1.9.0
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+service:
+  type: ClusterIP
+  ports:
+    tcp: 50001
+    ssl: 50002
+    ws: 50003
+    wss: 50004
+    stats: 8080
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # We recommend using the following values due to the resource intensive nature of bitcoind
+  # limits:
+  #   cpu: 3000m
+  #   memory: 5120Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 4096Mi
+
+persistence:
+  enabled: true
+  ## database data Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
+  accessMode: ReadWriteOnce
+  size: 200Gi
+
+bitcoindRpcPassSecretName: bitcoind-rpcpassword
+bitcoindRpcPort: 18443
```

```suggestion
bitcoindRpcPort: 8332
```
Should be mainnet port by default